aws/curator-ingress.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: extensions/v1beta1
```

Comment:
Can you tl;dr why this is used? No need to comment, just trying to grok.

Reply:
added a comment at the top, let me know if you want more details.